### PR TITLE
header changes

### PR DESCRIPTION
**Pull request by fszwedo**

### CODE REVIEW

File: frontend/src/components/PageHeader/PageHeader.module.css [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
     border: 1px solid #666666;
+    border-radius: 5px;
     padding: 1.5%;
     margin-bottom: 2%;
     background-color: #1C1C1C;
     padding-left: 3%;
     padding-right: 3%;
     color:#fff;
     text-align: left;
+    font-size: 1.5em;
```
Comment (robert-czwartosz) [inline frontend/src/components/PageHeader/PageHeader.module.css:11]: It would be better to set font-size in pixels, for example 30px.

**Review by robert-czwartosz — APPROVED** [review]
Looks good!
There are small fixes to do.

File: frontend/src/components/PageHeader/PageHeader.module.css (hunk 1 above)
Comment (fszwedo) [reply frontend/src/components/PageHeader/PageHeader.module.css]: switched to pixels

File: frontend/src/components/ReferenceProjects/ReferenceProjects.tsx [hunk 2]
```diff
@@ -79,7 +78,7 @@ const ReferenceProjects: React.FC< ReferenceProjectsProps > = props => {
                     color='primary'
                     onClick={handleAddButton} />
               </div>
-            </Box>
+            </Box>u
```
[the remark this answers is not in the document]
Comment (fszwedo) [reply frontend/src/components/ReferenceProjects/ReferenceProjects.tsx]: nicely spotted :) done

File: frontend/src/components/ReferenceProjects/ReferenceProjects.tsx [hunk 3]
```diff
@@ -67,9 +68,7 @@ const ReferenceProjects: React.FC< ReferenceProjectsProps > = props => {
   } else {
     return (
         <ThemeProvider theme={mainTheme}>
-          <Breadcrumbs aria-label="breadcrumb" color="primary" className={styles.breadcrumbs}>
-            <Typography color="primary">PROJECTS</Typography>
-          </Breadcrumbs>
+           <PageHeader name="Projects"></PageHeader>
```
[the remark this answers is not in the document]
Comment (fszwedo) [reply frontend/src/components/ReferenceProjects/ReferenceProjects.tsx]: done

**Comment by kjreczek** [comment]
Looks good. 👍

**Review by kjreczek — APPROVED** [review]

**Pull request merged**